helm/higress/templates/configmap.yaml

```diff
@@ -115,4 +115,16 @@ data:
 {{- include "mesh" . }}
 {{- end }}
 ---
-
+  {{- if .Values.gateway.customBootstrap.enabled  }}
```

Comment:
It is recommended to use `enableSkywalking` to control this logic, which is easier to understand.